piff/input.py

```diff
@@ -176,422 +80,558 @@ def makeStars(self, logger=None):
+
+            :chipnum:       The id number of this chip used to reference this image [default:
+                            image_num]
+
+            :image_hdu:     The hdu to use in the image files. [default: None, which means use
+                            either 0 or 1 as typical given the compression sceme of the file]
+            :weight_hdu:    The hdu to use for weight images. [default: None, which means a weight
+                            image with all 1's will be automatically created]
+            :badpix_hdu:    The hdu to use for badpix images. Pixels with badpix != 0 will be given
+                            weight == 0. [default: None]
+            :noise:         Rather than a weight image, provide the noise variance in the image.
+                            (Useful for simulations where this is a known value.) [default: None]
+
+            :cat_hdu:       The hdu to use in the catalog files. [default: 1]
+            :x_col:         The name of the X column in the input catalogs. [default: 'x']
+            :y_col:         The name of the Y column in the input catalogs. [default: 'y']
+            :flag_col:      The name of a flag column in the input catalogs.  Anything with
+                            flag != 0 is removed from the catalogs. [default: None]
+            :use_col:       The name of a use column in the input catalogs.  Anything with
+                            use == 0 is removed from the catalogs. [default: None]
+            :sky_col:       The name of a column with sky values. [default: None]
+            :gain_col:      The name of a column with gain values. [default: None]
+            :sky:           The sky level to subtract from the image values. [default: None]
+                            Note: It is an error to specify both sky and sky_col. If both are None,
+                            no sky level will be subtracted off.
+            :gain:          The gain to use for adding Poisson noise to the weight map.  [default:
+                            None] It is an error for both gain and gain_col to be specified.
+                            If both are None, then no additional noise will be added to account
+                            for the Poisson noise from the galaxy flux.
+            :minsnr:        The minimum S/N ratio to use.  If an input star is too faint, it is
+                            removed from the input list of PSF stars.
+            :maxsnr:        The maximum S/N ratio to allow for any given star.  If an input star
+                            is too bright, it can have too large an influence on the interpolation,
+                            so this parameter limits the effective S/N of any single star.
+                            Basically, it adds noise to bright stars to lower their S/N down to
+                            this value.  [default: 100]
+            :nstars:        Stop reading the input file at this many stars. [default: None]
```

Comment:
It's per input star catalog.  So typically per ccd.  I'll add some documentation to make that clearer.